website/docs/r/aws_secret_backend.html.md

```diff
@@ -67,6 +67,17 @@ for credentials issued by this backend.
 
 * `sts_endpoint` - (Optional) Specifies a custom HTTP STS endpoint to use.
 
+* `username_template` - (Optional)  Template describing how dynamic usernames are generated. The username template is used to generate both IAM usernames (capped at 64 characters) and STS usernames (capped at 32 characters). If no template is provided the field defaults to the template:
+
+```sh
```

Comment:
We can remove the `sh` 😄

Reply:
markdown linter in my IDE yelled at me, will change.

Reply:
removed.